### PR TITLE
Update .NET profiler doc on Nuget setup

### PR DESCRIPTION
**Pull request by gleocadie**
### What does this PR do?
This PR update the Nuget-based setup for the .NET profiler

### Motivation

Customer reported that the doc did not help and we figured out that it was out-of-date or missing important information

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/profiler/enabling/dotnet.md [hunk 1]
```diff
@@ -299,13 +299,13 @@ To install the .NET Profiler per-application:
...
-   Linux ARM64      | `<APP_DIRECTORY>/datadog/linux-arm64/Datadog.Trace.ClrProfiler.Native.so`
-   Windows x64      | `<APP_DIRECTORY>\datadog\win-x64\Datadog.Trace.ClrProfiler.Native.dll`
-   Windows x86      | `<APP_DIRECTORY>\datadog\win-x86\Datadog.Trace.ClrProfiler.Native.dll`
+   Operating System and Process Architecture | CORECLR_PROFILER_PATH Value | LD_PRELOAD Value
+   ------------------------------------------|-----------------------------------------------------------------
+   Alpine Linux x64 | `<APP_DIRECTORY>/datadog/linux-musl-x64/Datadog.Trace.ClrProfiler.Native.so`| <APP_DIRECTORY>/datadog/linux-musl-x64/Datadog.Linux.ApiWrapper.x64.so
+   Linux x64        | `<APP_DIRECTORY>/datadog/linux-x64/Datadog.Trace.ClrProfiler.Native.so` | <APP_DIRECTORY>/datadog/linux-x64/Datadog.Linux.ApiWrapper.x64.so
+   Linux ARM64      | `<APP_DIRECTORY>/datadog/linux-arm64/Datadog.Trace.ClrProfiler.Native.so`| <APP_DIRECTORY>/datadog/linux-arm64/Datadog.Linux.ApiWrapper.x64.so
+   Windows x64      | `<APP_DIRECTORY>\datadog\win-x64\Datadog.Trace.ClrProfiler.Native.dll` | N/A
+   Windows x86      | `<APP_DIRECTORY>\datadog\win-x86\Datadog.Trace.ClrProfiler.Native.dll` | N/A
```
Comment (buraizu) [inline content/en/profiler/enabling/dotnet.md:308]:
```suggestion
   Operating System and Process Architecture | CORECLR_PROFILER_PATH Value | LD_PRELOAD Value
   ------------------------------------------|-------------------------------------|---------------------------
   Alpine Linux x64 | `<APP_DIRECTORY>/datadog/linux-musl-x64/Datadog.Trace.ClrProfiler.Native.so`| `<APP_DIRECTORY>/datadog/linux-musl-x64/Datadog.Linux.ApiWrapper.x64.so`
   Linux x64        | `<APP_DIRECTORY>/datadog/linux-x64/Datadog.Trace.ClrProfiler.Native.so` | `<APP_DIRECTORY>/datadog/linux-x64/Datadog.Linux.ApiWrapper.x64.so`
   Linux ARM64      | `<APP_DIRECTORY>/datadog/linux-arm64/Datadog.Trace.ClrProfiler.Native.so`| `<APP_DIRECTORY>/datadog/linux-arm64/Datadog.Linux.ApiWrapper.x64.so`
   Windows x64      | `<APP_DIRECTORY>\datadog\win-x64\Datadog.Trace.ClrProfiler.Native.dll` | N/A
   Windows x86      | `<APP_DIRECTORY>\datadog\win-x86\Datadog.Trace.ClrProfiler.Native.dll` | N/A
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, but the table formatting is currently broken. The suggestion I added seemed to resolve it in a local build ([screenshot](https://a.cl.ly/NQu4B71m)), although the heading of the column on the left seems a bit too long for its space.

**Comment by gleocadie** [comment]
> Thanks for the PR, but the table formatting is currently broken. The suggestion I added seemed to resolve it in a local build ([screenshot](https://a.cl.ly/NQu4B71m)), although the heading of the column on the left seems a bit too long for its space.

Thanks @buraizu

**Review by buraizu — APPROVED** [review]
Thanks, looks like the table is rendering as expected, although I can see that it's overflowing its bounds slightly ([screenshot](https://a.cl.ly/WnuZ849O)). In the interest of getting this information to our users I'll go ahead and merge as it is, but I've created DOCS-4737 to explore other options for rendering this data that won't overflow, such as a description list.

**Pull request merged**